contentcuration/contentcuration/__init__.py

```diff
@@ -2,4 +2,3 @@
 
 # This will make sure the app is always imported when
 # Django starts so that shared_task will use this app.
-from .celery import app as celery_app
```

Comment:
@ralphiee22 @micahscopes  Was there any reason for removing this line here: it seems all the celery tasks are broken without this now. My initial debugging indicates that just putting that makes celery work again, so motion to put it back in...